scripts/irods/test/test_ichksum.py

```diff
@@ -68,11 +72,18 @@ def test_ichksum_force(self):
         file_chksum = lib.file_digest(filename, 'sha256', encoding='base64')
         self.admin.assert_icommand(['iput', filename])
         self.admin.assert_icommand(['ichksum', filename], 'STDOUT_SINGLELINE', file_chksum)
+
         # Mess with file in vault
         full_path = os.path.join(self.admin.get_vault_session_path(), filename)
         lib.cat(full_path, 'XXXXX')
         new_chksum = lib.file_digest(full_path, 'sha256', encoding='base64')
-        self.admin.assert_icommand(['ichksum', '-f', filename], 'STDOUT_SINGLELINE', new_chksum)
+
+        # Show that the new checksum does not match the original.
+        self.assertNotEqual(file_chksum, new_chksum)
+
+        # The checksum will remain the same because ichksum reads, at most, the number of bytes
+        # recorded in the catalog.
+        self.admin.assert_icommand(['ichksum', '-f', filename], 'STDOUT_SINGLELINE', file_chksum)
```

Comment:
Does this test still make sense? As in does the behavior match the test's name?

Reply:
right, we now trust the catalog, so this is a moot test.

it would behave the same, with and without the `-f` in this case, correct?

Reply:
Correct. This PR adds two tests that demonstrate this behavior.

Will remove this test.

Reply:
Test removed.